haskell/src/Indigo/Contracts/Safelist.hs

```diff
@@ -0,0 +1,70 @@
+-- SPDX-FileCopyrightText: 2020 tqtezos
+-- SPDX-License-Identifier: MIT
+
+{-# OPTIONS_GHC -Wno-orphans #-}
+module Indigo.Contracts.Safelist
+  ( Parameter
+  , Storage (..)
+  , mkStorage
+  , safelistContract
+  ) where
+
+import Data.Set (fromList)
+
+import Indigo as Indigo
+import Lorentz.Run (Contract)
+
+data Storage = Storage
+  { sTransfers :: [(Address, Address)]
```

Comment:
Why not `Set`? I suppose you copied it from globacap. When it was written there, `morley` didn't support pairs in sets, but now it does.

Also I see that https://github.com/serokell/tezos-globacap/blob/master/src/Indigo/Contracts/Safelist.hs is proprietary, so it's illegal to copy-paste it here, but I believe we can do something about it. I've asked Globacap people whether they want to add a license there.

Reply:
I have changed it to Set.

Reply:
> I've asked Globacap people whether they want to add a license there.

They are discussing it. I think we can pretend it's fine for now and just merge that code, hoping they will decide to add a permissive license. If they don't do it, we will get back to it later.